shm/inc/alcohol.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "cargo.hpp"
+
+namespace {
+constexpr size_t BASE_POWER = 96;

```

Comment:
Personally I would change this for `constexpr auto kBaseAlcoholConcentration = 0.96f` but it's my opinion :)

Reply:
I think that Alcohol concentration should be represented as a percentage rather than fraction. That's more natural. I've changed it to float to give it more flexibility so it doesn't need conversion during price counting.

Reply:
Okay, I see your point of view :) You can change the name of the constexpr variable to be consistent with other constexpr names. Please refer to map.cpp i.e. I think that style convention suggested by you is rather reserved for Macros.

Reply:
Ok you're right. I haven't checked out convention about naming constexpresions. I will change it.  That was force of habits :D